code-studio/src/js/progress.js

```diff
@@ -134,7 +134,10 @@ progress.renderCourseProgress = function (scriptData) {
   var store = loadProgress(scriptData);
   var mountPoint = document.createElement('div');
 
-  $.ajax('/api/user_progress/' + scriptData.name).done(data => {
+  $.ajax(
+    '/api/user_progress/' + scriptData.name,
+    { data: { user_id: clientState.queryParams('user_id') } }
```

Comment:
when clientState.queryParams('user_id')  is undefined, no parameter gets added to the request url.